src/api/resources/project.py

```diff
@@ -101,7 +130,7 @@ def get_num_sample_units(self, obj):
         return num_sample_units
 
 
-class ProjectSerializer(DynamicFieldsMixin, BaseProjectSerializer):
```

Comment:
is DynamicFieldsMixin not needed anymore?

Reply:
@gridcell I moved it to `BaseProjectSerializer` (i.e. up in the inheritance chain). Does that work?